psl/psl.go

```diff
@@ -0,0 +1,155 @@
+// package psl defines the Psl struct containing alignment data as well as functions and methods that operate on the Psl struct
+package psl
+
+import (
+	"github.com/vertgenlab/gonomics/common"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+	"strings"
+)
+
+// Psl struct declares data fields and types for Psl struct.
```

Comment:
Sort of a side note, but Craig and I were discussing how LiftOver can use a PSL instead of a chain file for matching positions in the old and new assembly. I suppose this might be theoretically possible with Axt and MultiFa as well. perhaps we could consider some sort of "alignment" interface between these formats in the future